ksp_plugin/plugin.hpp

```diff
@@ -427,8 +429,15 @@ class Plugin {
           physics::KeplerianElements<Barycentric>> const& keplerian_elements,
       MassiveBody const& body);
 
+  void AddPart(not_null<Vessel*> vessel,
```

Comment:
Comment: Adds a part to a vessel, recording it in the appropriate map and setting up a deletion callback.